tests/test_server.py

```diff
@@ -287,6 +287,15 @@ def test_server_downloadDatabases(tmpdir, plex):
     assert len(tmpdir.listdir()) > 1
 
 
+def test_server_browse(plex, movies):
+    # browse root
+    paths = plex.browse()
+    assert len(paths)
+    # browse of the files of the movie lib.
+    for path, paths, files in plex.walk(movies.locations[0]):
+        assert len(files)
```

Comment:
to account for subfolders
`assert len(files) if files else (path and len(paths))`

Reply:
The reason why i only wanted to test for the files is because i don't know what file structure they have choosen.

Reply:
This will fail if there are no files in the root of the movie library folder (i.e. `/media/Movies`, but all files are in subfolders `/media/Movies/Big Buck Bunny(2008)/Big Buck Bunny (2008).mkv`).